src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -93,38 +97,44 @@ public void pathPlannerInit() {
     public void periodic() {
         String state = "";
         
-        double xMovement = MathUtil.applyDeadband(-robot.controller.getLeftY(), Constants.stickDeadband);
-        double rotation = MathUtil.applyDeadband(-robot.controller.getRightX(), Constants.stickDeadband);
-        double yMovement = MathUtil.applyDeadband(robot.controller.getLeftX() * Constants.Drive.leftXSign, Constants.stickDeadband);
+        double xMovement = MathUtil.applyDeadband(-robot.controller.getLeftY(), Constants.STICK_DEADBAND);
+        double rotation = MathUtil.applyDeadband(-robot.controller.getRightX(), Constants.STICK_DEADBAND);
+        double yMovement = MathUtil.applyDeadband(robot.controller.getLeftX() * Constants.Drive.leftXSign, Constants.STICK_DEADBAND);
 
 
         if (driveStates == DriveStates.FIELD_ABSOLUTE) {
             state = "Field Absolute";
             fieldRelative = false;
 
-            if (robot.controller.getBButtonPressed()) {
+            if (robot.controller.getXButtonPressed()) {
                 driveStates = DriveStates.FIELD_RELATIVE;
                 System.out.println("FIELD relative ON");
             }
-
         } else if (driveStates == DriveStates.FIELD_RELATIVE) {
             state = "Field Relative";
             fieldRelative = true;
 
-            if (robot.controller.getBButtonPressed()) {
+            if (robot.controller.getXButtonPressed()) {
                 driveStates = DriveStates.FIELD_ABSOLUTE;
                 System.out.println("FIELD Relative OFF");
             }
-            
-        } 
-        
-        if (robot.controller.getXButtonPressed()) {
+        }
+
+        if (robot.controller.getStartButtonPressed()) {
             swerveDrive.zeroGyro();
             System.out.println("Gyro Zeroed");
+        } else if (robot.controller.getYButton()) {
+            swerveDrive.lockPose();
+        } else if (robot.controller.getLeftBumper()) {
+            xMovement = MathUtil.applyDeadband(-robot.controller.getLeftY() * Constants.Drive.JOYSTICK_VALUE_MULTIPLIER, Constants.STICK_DEADBAND);
```

Comment:
rename constant from JOYSTICK_VALUE_MULTIPLIER to SLOWMODE_TRANSLATION_MULTIPLIER or something more descriptive